NuGet.Docs/Create/Creating-and-Publishing-a-Package.md

```diff
@@ -240,7 +240,8 @@ When a solution-level package is installed, it is tracked in a packages.config
 file in the .nuget directory, rather than in a packages.config file in a
 specific project.
 
-Note: Solution level packages are not supported in NuGet 3.0.0-3.2.0, and you will need to install such packages into a common project
+Note: Solution level packages are only supported in NuGet 3.0.0-3.3.0, and you will need to install such packages into a common project. 
+Content items will only be delivered for use in projects that are managed with a packages.config file.
```

Comment:
How is content related here? is that a separate note? if so can we point to the new alternative for projects using project.json (starting with nuget 3.3) ?